youtube_dl/extractor/tubitv.py

```diff
@@ -13,38 +13,29 @@
 
 
 class TubiTvIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?tubitv\.com/(?:video|movies|tv-shows)/(?P<id>[0-9]+)'
+    _VALID_URL = r'https?://(?:www\.)?tubitv\.com/(?:video|movies|tv-shows|live)/(?P<id>[0-9]+)'
     _LOGIN_URL = 'http://tubitv.com/login'
     _NETRC_MACHINE = 'tubitv'
     _GEO_COUNTRIES = ['US']
     _TESTS = [{
-        'url': 'http://tubitv.com/video/283829/the_comedian_at_the_friday',
-        'md5': '43ac06be9326f41912dc64ccf7a80320',
+        'url': 'https://tubitv.com/movies/229877/carnival-of-souls',
+        'md5': '275a9f78428cfd6428520989408bcfe9',
         'info_dict': {
-            'id': '283829',
+            'id': '229877',
             'ext': 'mp4',
-            'title': 'The Comedian at The Friday',
-            'description': 'A stand up comedian is forced to look at the decisions in his life while on a one week trip to the west coast.',
-            'uploader_id': 'bc168bee0d18dd1cb3b86c68706ab434',
+            'title': 'Carnival of Souls',
+            'description': 'After a traumatic accident, a woman becomes drawn to a mysterious abandoned carnival.',
+            'uploader_id': 'da2522b1c94be4a578da6ba674925159',
         },
     }, {
-        'url': 'http://tubitv.com/tv-shows/321886/s01_e01_on_nom_stories',
-        'only_matching': True,
-    }, {
-        'url': 'http://tubitv.com/movies/383676/tracker',
-        'only_matching': True,
-    }, {
-        'url': 'https://tubitv.com/movies/560057/penitentiary?start=true',
+        'url': 'https://tubitv.com/live/715943/wb-tv-watchlist',
         'info_dict': {
-            'id': '560057',
+            'id': '715943',
             'ext': 'mp4',
-            'title': 'Penitentiary',
-            'description': 'md5:8d2fc793a93cc1575ff426fdcb8dd3f9',
-            'uploader_id': 'd8fed30d4f24fcb22ec294421b9defc2',
-            'release_year': 1979,
-        },
-        'params': {
-            'skip_download': True,
+            'title': 'WB TV Watchlist',
+            'description': 're:.*',
```

Comment:
The `description` field was required for the test, but the current value of that field ("Catch up on fan favorite series that are a must for your watch list – like Westworld, Raised by Wolves, Third Watch and more.") implies that it might be frequently updated.

Reply:
Just use `compat_str` as the test value:
```py
            'description': compat_str,
```

Reply:
Ah, I misinterpreted the documentation surrounding that. Updated.